main.go

```diff
@@ -9,24 +9,27 @@ import (
 	"strings"
 	"time"
 
+	"github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/session"
+
```

Comment:
remove empty lines from import list

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/615bd5f8c8369ebd51f43372703021968554ac84